docs/kubernetes_deploy.md

```diff
@@ -130,12 +182,12 @@ REVISION: 1
 TEST SUITE: None
 ```
 
-Check that the model analyzer pod is running and that it has two containers.
+Check that the model analyzer pod is running.
```

Comment:
Why did it use to require to containers? Is that one of them was used for Triton Server and the other one was used for Triton Server?

Reply:
Yes